studio-api/components/MongoMigration/version/1.6.2/conversationGenerations.js

```diff
@@ -0,0 +1,67 @@
+const debug = require("debug")(
+  "linto:migration:1.6.2:conversationGenerations",
+)
+
+/**
+ * Migration: Create conversationGenerations collection
+ *
+ * This collection tracks generation history per conversation + service combination.
+ * Each generation maps to a jobId in LLM Gateway.
+ */
+module.exports = {
+  async up(db) {
+    debug("Creating conversationGenerations collection and indexes")
+
+    // Check if collection exists
+    const collections = await db.listCollections({ name: "conversationGenerations" }).toArray()
+    if (collections.length === 0) {
+      await db.createCollection("conversationGenerations")
+      debug("Created conversationGenerations collection")
+    }
+
+    const collection = db.collection("conversationGenerations")
+
+    // Create indexes
+    // Compound index for listing generations by conversation and service
+    await collection.createIndex(
```

Comment:
Replace duplicated collection creation and dropping logic with the existing functions in `components/MongoMigration/controllers/migration/`.  

Use `await initDb(db, collectionName)` and `await dropDb(db, collections_name) `instead of reimplementing the logic.